packages/frontmatter/src/downloads.tsx

```diff
@@ -130,9 +130,9 @@ export function DownloadsDropdown({ exports }: HasExports) {
   if (!exports || exports.length === 0) return null;
   return (
     <Menu as="div" className="relative flex inline-block mx-1 grow-0">
-      <Menu.Button className="relative ml-2 -mr-1">
+      <Menu.Button className="relative ml-2 -mr-1 hover:text-[#E18435]">
```

Comment:
I don't think that we should introduce this colour here as it affects all downstream themes with a very opinionated styling. 

I realise that this is in use on the LaunchButton already (which was introduced much later than downloads) but I would say that having that there also is too opinionated - in both cases I think we should be passing in a style/class in as props, and then introducing these styles in book or article theme when the component is used.

My motivation is that `packages/*` should have little / no opinionated styling while, the core `themes/*` can apply opinionated styling.